fortio/stage/cert.yaml

```diff
@@ -0,0 +1,101 @@
+# TODO: parametrize the parent directory yaml and/or keep in sync when
+# succesfully experimenting here / on -stage sites
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: letsencrypt-stage
```

Comment:
Why do these need different names if they're created in different clusters? It looks like the only difference is the host names? if so, you could use something like `envsubst` with `${FORTIO_ISTIO_IO}` and ${ISTIO_FORTIO_ORG} as a quick-and-dirty templating solution to avoid duplicating the entire set of yamls.

Reply:
I am a fierce enemy of copy/pasta too, and I have that in the TODO but the flip side of the argument is I can experiment on stage/ without risking prod - which I started to do, added grpc and the new -base-url only in stage/ so far

(yet for the cert.yaml specifically, yes just the domain list is the difference)

Reply:
ps: cool I didn't know about `envsubst` specifically... could be a replacement for updateVersion/sed/helm templates/... how standard is it?

Reply:
@pipejakob pointed it out to me. Looks likes its `getext` package which is easily available on Linux and osx (via brew). I used it [here](https://github.com/istio/istio/blob/master/install/kubernetes/webhook-patch-ca-bundle.sh) to patch the webhook ca-bundle but hedged and added a fallback with `sed incase `envsubst` wasn't available.